src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        return UserService.findByEmail(email)
+                .getPassword()
+                .equals(password);
```

Comment:
you will get NPE after refactoring UserService, so you'll need to make changes here too